app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,64 @@
+class AlbumsController < ApplicationController
+  def get_album
+    id = params[:id]
+    @album = Album.find(id)
+  end
+
+  def index
+    @albums = Album.all
```

Comment:
It looks like you are not using the instance variable `@albums` in your index view. This means you do not need to expose it to the view layer and a local variable like `albums` would suffice.